### PR TITLE
doc(features): Adjust feature `README.md` files

### LINKED ISSUE #1399: Align all features `README.md` files to a common template
**What would you like to be added**:
We should align all features `REAMDE.md` files to a common template. Currently, they're missing information, have different layouts or are just empty. Next to this, this may be needed when the new Garden Linux website (see also #317) is auto generated by website tags.

Given by the currently used ones and the upcoming website we might use the following template structure:

```
## Feature: $featurename
### Description
<website-feature>
A short description of the feature. Why would it be used and in which cases.
</website-feature>

### Features
A briefly overview of changes that are provided by this feature or additional notes how to use and change things to the operators needs.

### Unit testing
A short overview of unit tests. Are they consumed by all fixtures or on dedicated ones?

### Meta
|||
|---|---|
|type|$platform/$feature|
|artifact|`$output_type`|
|included_features|$list|
|excluded_features|$list|
```

Affects:
- [x] _build
- [x] _dev
- [x] _ignite
- [x] _iso
- [x] _nopkg
- [x] _prod
- [x] _pxe
- [x] _readonly
- [x] _secureboot
- [x] _slim
- [x] aide
- [x] ali
- [x] aws
- [x] azure
- [x] base
- [x] chost
- [x] cis
- [x] cisAudit
- [x] cisModprobe
- [x] cisOS
- [x] cisPackages
- [x] cisPartition
- [x] cisSshd
- [x] cisSysctl
- [x] clamav
- [x] cloud
- [x] container
- [x] example
- [x] fedramp
- [x] firecracker
- [x] firewall
- [x] gardener
- [x] gcp
- [x] githubActionRunner
- [x] khost
- [x] kvm
- [x] metal
- [x] oci
- [x] openstack
- [x] pkg.exclude
- [x] server
- [x] vhost
- [x] vmware

### PR DESCRIPTION
**Pull request by gyptazy**
doc(features): Adjust feature `README.md` files

**How to categorize this PR?**

/kind enhancement
/area os
/os garden-linux

**What this PR does / why we need it**:
Adjust feature `README.md` files

**Which issue(s) this PR fixes**:
Fixes #1399

**Special notes for your reviewer**:

**Release note**:

```feature user
NONE
```

### CODE REVIEW

**Review by Vincinator — COMMENTED** [review]
Skipped some features, will add more feedback later.
In general this looks like an improvement, thank you!

File: features/cloud/README.md [hunk 1]
```diff
@@ -1,9 +1,19 @@
-## Cloud Feature: cloud
-<website-feature> cloud functionality (cloud kernel, cloud boot) </website-feature>
+## Feature: cloud
+### Description
```
[the remark this answers is not in the document]
Comment (gyptazy) [reply features/cloud/README.md:2]: @Vincinator, good hint. Added, thanks!

**Comment by gyptazy** [comment]
@Vincinator, I just (re)moved the docs for:

### oci
`oci` (element feature) gets deprecated by `_oci` (flag feature) in PR https://github.com/gardenlinux/gardenlinux/pull/1309.
Therefore, the docs within the PR have been updated by commit: https://github.com/gardenlinux/gardenlinux/pull/1309/commits/021497ecb0f043820f33ddc2f9db3d99654bb604

### server
`server` will be slightly adjusted where `ssh` for the opens server will be a dedicated feature in PR https://github.com/gardenlinux/gardenlinux/pull/1351.
Therefore, the docs within the PR have been updated by commit: https://github.com/gardenlinux/gardenlinux/pull/1351/commits/a0b1ee2b428dcd116e44002cc680abecf0ab5772

This will update the docs as soon as the related feature gets merged and won't bring up any merge conflicts. jfyi

**Review by Vincinator — APPROVED** [review]
Thanks, this looks good to me

**Comment by Vincinator** [comment]
It is save to merge, despite the failed amd64/kvm build pipeline. This PR only changes documentation, and the error was about network connectivity.
```
=================================== FAILURES ===================================
__________________________________ test_tiger __________________________________

client = <helper.sshclient.RemoteClient object at 0x7f6051fd8b20>
testconfig = {'features': ['_slim', 'base', 'server', 'cloud', 'kvm'], 'image': '/gardenlinux/.build//kvm-amd64-today-4813def.tar.xz', 'ip': '127.0.0.1', 'port': 2222, ...}
chroot = None

    def tiger(client, testconfig, chroot):
        """Run tiger to test for system security vulnerabilities"""
        utils.AptUpdate(client)
        (exit_code, output, error) = client.execute_command(
            "apt-get install -y --no-install-recommends tiger",
            quiet=True)
>       assert exit_code == 0, f"no {error=} expected"
E       AssertionError: no error='E: Failed to fetch https://deb.debian.org/debian/pool/main/t/tiger/tiger_3.2.4%7erc1-3.2_amd64.deb  Cannot initiate the connection to debian.map.fastly.net:443 (2a04:4e42:62::644). - connect (101: Network is unreachable) Could not connect to debian.map.fastly.net:443 (199.232.138.132), connection timed out Cannot initiate the connection to deb.debian.org:443 (2a04:4e42:62::644). - connect (101: Network is unreachable)\nE: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?\n' expected

chroot     = None
client     = <helper.sshclient.RemoteClient object at 0x7f6051fd8b20>
error      = 'E: Failed to fetch https://deb.debian.org/debian/pool/main/t/tiger/tiger_3.2.4%7erc1-3.2_amd64.deb  Cannot initiate t...(101: Network is unreachable)\nE: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?\n'
exit_code  = 100
output     = 'Reading package lists...\nBuilding dependency tree...\nReading state information...\nSuggested packages:\n  lynis\nRe...ut Cannot initiate the connection to deb.debian.org:443 (2a04:4e42:62::644). - connect (101: Network is unreachable)\n'
testconfig = {'features': ['_slim', 'base', 'server', 'cloud', 'kvm'], 'image': '/gardenlinux/.build//kvm-amd64-today-4813def.tar.xz', 'ip': '127.0.0.1', 'port': 2222, ...}

helper/tests/tiger.py:13: AssertionError
```

**Pull request merged**